# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/linux/linux26/72-pci-h_mod_devicetable-remove.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./include/linux/pci.h.orig	2004-08-25 05:39:31.000000000 +0200
+++ ./include/linux/pci.h	2004-10-09 17:22:44.049099544 +0200
@@ -17,7 +17,9 @@
 #ifndef LINUX_PCI_H
 #define LINUX_PCI_H
 
+#ifdef __KERNEL__
 #include <linux/mod_devicetable.h>
+#endif
 
 /*
  * Under PCI, each device has 256 bytes of configuration address space,
